python/ray/rllib/agents/dqn/dqn_policy_graph.py

```diff
@@ -492,7 +495,9 @@ def extra_compute_action_feed_dict(self):
     def extra_compute_grad_fetches(self):
         return {
             "td_error": self.loss.td_error,
-            LEARNER_STATS_KEY: self.loss.stats,
+            LEARNER_STATS_KEY: dict({
+                "cur_lr": tf.cast(self.cur_lr, tf.float64),
```

Comment:
Oh, this might be it: this is creating a new tf operation each time a gradient is computed, which may not be garbage collected by TensorFlow. The cast op should be created in the __init__ of the class.

Reply:
Yes you are right, thanks! I pushed a fix for it.